examples/src/main/java/com/teamdev/jxbrowser/examples/PrintFromJava.java

```diff
@@ -65,7 +64,8 @@ public void windowClosing(WindowEvent e) {
             frame.setLocationRelativeTo(null);
             frame.setVisible(true);
 
-            browser.navigation().loadUrl("https://www.google.com");
+            browser.navigation()
+                   .loadUrl("https://webglsamples.org/dynamic-cubemap/dynamic-cubemap.html");
```

Comment:
For Print Preview example it's better to load a text page. Let's load https://en.wikipedia.org/wiki/Printing